packages/flutter_tools/test/src/context.dart

```diff
@@ -382,14 +383,6 @@ class MockHttpClient extends Mock implements HttpClient {}
 
 class MockCrashReporter extends Mock implements CrashReporter {}
 
-class FakePlistParser implements PlistParser {
```

Comment:
I changed my `TestPlistParser` to `FakePlistParser`, and replaced the one in the context overrides.